src/components/resumes/Resumes.jsx

```diff
@@ -30,28 +18,24 @@ function ResumeList() {
   }, []);
 
   return (
-    <div className="Resumes-Container">
-      <Button color="secondary" variant="outlined">
-        Create Resume
-      </Button>
-      <div className="Resume-List">
-        <TableContainer component={Paper}>
-          <Table aria-label="simple table" className={classes.table}>
-            <TableBody>
-              {resumes.length > 0
-                ? resumes.map((resume) => (
-                    <TableRow key={resume.id}>
-                      <TableCell component="th" scope="row">
-                        <Box {...resume} />
-                      </TableCell>
-                    </TableRow>
-                  ))
-                : null}
-            </TableBody>
-          </Table>
-        </TableContainer>
+    <Container className="Resumes-Container">
+      <div className="Button-Container">
+        <Link to="/resumes/entry">
+          <Button color="primary" variant="contained">
+            New Resume
+          </Button>
+        </Link>
+      </div>
+      <div className="Resumes-List" key={resumes.id}>
+        {resumes.length > 0
+          ? resumes.map((resume) => (
+              <div className="Resume-Box" key={resume.id}>
```

Comment:
I think we are already using lower case for our css classes. 
e.g.  resumes-container instead of Resumes-Container
Please re-write them.

Reply:
ohh noo!! ok I'll fix.